src/FormBuilder/Routes/RegisterFormBuilderPageRoute.php

```diff
@@ -80,13 +82,25 @@ public function renderPage()
 
         $formBuilderStorage->loadInFooter()->enqueue();
 
+        $builderPaymentGatewayData = array_map(function ($gateway) {
+            return [
+                'id' => $gateway::id(),
+                'label' => give_get_gateway_checkout_label($gateway::id()) ?? $gateway->getPaymentMethodLabel(),
+                'supportsSubscriptions' => $gateway->supportsSubscriptions(),
+            ];
+        }, give(DonationFormRepository::class)->getEnabledPaymentGateways($donationFormId));
```

Comment:
Remember we also need to filter these gateways by next gen compatibility or _all_ gateways will be loaded. We should be able to just check if they implement `NextGenPaymentGatewayInterface`

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/140/commits/3ab5ba6f2e9690ee68b62160d72d3ad65582ffcf.